backend/module/eCampCore/src/EntityService/CategoryService.php

```diff
@@ -86,6 +88,24 @@ function ($contentType) use ($category) {
         return $category;
     }
 
+    protected function createEntityPost(BaseEntity $entity, $data): BaseEntity {
+        /** @var Category $category */
+        $category = parent::createEntityPost($entity, $data);
+
+        if (!isset($data->createRootContentNode) || $data->createRootContentNode) {
+            $q = $this->findCollectionQueryBuilder(ContentType::class, 'c', null)->where("c.name = 'ColumnLayout'");
+            $columnLayout = $this->getQuerySingleResult($q);
```

Comment:
Bin noch mässig happy hiermit.
* Wäre ein Repository oder eine Methode auf dem ContentNodeService geeignet dafür?
* Könnte man den String `'ColumnLayout'` in eine Konstante auslagern, sodass wir das in Zukunft vielleicht irgendwann mal konfigurierbar machen könnten (vielleicht auch pro Camp statt global, weiss noch nicht)?
* Wäre es clever wenn wir trotzdem einen parameterisierten Query machen, falls das in Zukunft mal jemand flexibler macht sodass man auch andere ContentTypes nach Namen finden kann?

Reply:
Etwas eleganter:
`$contentNodeRepository->findOneBy('name' => 'ColumnLayout')`

und `ContentNodeRepository` kann via DI in den `CategoryService` gekippt werden.

Konfiguriere finde ich gut. Am Besten in `eCamp\CoreData\Prod\ContentTypeData`, dann ist sichergestellt, dass die Seed-Daten gleich sind, wie der hardcoded name.

Reply:
- [x] ContentNodeService erlaubt erstellen von ContentNode anhand von ContentTypeName
- [x] ContentType wird via Repository geladen
- [x] 'ColumnLayout' als Konstante erfasst (kann bei Bedarf später verschoben werden; aktuell aber noch unklar wohin. Config? Camp? usw...)